styleguide/backstop_data/ci_report/xunit.xml

```diff
@@ -1,2 +1,2 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuites><testsuite name="BackstopJS" tests="15"><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/></testsuite></testsuites>
\ No newline at end of file
+<testsuites><testsuite name="BackstopJS" tests="15"><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-button" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› atoms-subtitle" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/><testcase name=" ›› press-release-page" classname="#sg-viewport"/></testsuite></testsuites>
```

Comment:
Remove this file from being committed. It should be updated when you merge with `develop` and resolve merge conflicts however if it isn't you can run `git rm -r --cached styleguide/backstop_data/ci_report/` to get git to stop detecting the changes.